iceoryx_examples/icecrystal/Readme.md

```diff
@@ -12,7 +12,9 @@ memory.
 We re-use the binaries from
 [icedelivery](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_examples/icedelivery).
 
-[![asciicast](https://asciinema.org/a/407442.svg)](https://asciinema.org/a/407442)
+<!-- @todo Add expected output with asciinema recording before v2.0-->
+<!-- The current one is outdated, e.g. the the ServiceDiscovery publisher is missing -->
+<!-- [![asciicast](https://asciinema.org/a/407442.svg)](https://asciinema.org/a/407442) -->
```

Comment:
@mossmaurice Will record the asciicast for this and other examples after everything is validated, see #743.